src/settings/toml/tests/deploy_config.rs

```diff
@@ -216,7 +216,7 @@ fn it_can_get_a_multi_route_zoned_deploy_config() {
         .iter()
         .map(|p| Route {
             script: Some(script_name.to_string()),
-            pattern: p.to_string(),
+            pattern: (*p).to_string(),
```

Comment:
these are all `&&str` and it's faster to dereference them before calling `.to_string()`